apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/template-description.tsx

```diff
@@ -62,12 +62,82 @@ export const TemplateDescription: FunctionComponent<TemplateDescriptionPropsInte
     } = props;
 
     const { t } = useTranslation();
+    const { getLink } = useDocumentation();
 
+    const resolveDocumentationLink = (): ReactElement => {
+        const templateName = template?.name;
+
+        if ( templateName === "User-Age-Based" ) {
```

Comment:
Check other places as well.
```suggestion
        if (templateName === "User-Age-Based") {
```